### PR TITLE
fix: drop incorrect versions from compatibility.json 

### PR DESCRIPTION
**Pull request by smallstepman**
not sure if we should add 0.14.4 @ericswanson-dfinity ?

### CODE REVIEW

**Comment by ericswanson-dfinity** [comment]
> not sure if we should add 0.14.4 @ericswanson-dfinity ?

Should have the same compatibility as 0.14.3, yes?

File: compatibility.json [hunk 1]
```diff
@@ -1,26 +1,26 @@
 {
   "0.15.0": {
       "nns": {
-          "versions": ["0.1.0", "0.2.0"]
+          "versions": ["0.2.0"]
```
Comment (ericswanson-dfinity) [inline compatibility.json:4]: as of this moment, the release-0.15.0 branch does not support the new filenames

Comment (smallstepman) [reply compatibility.json:4]: should we first update release-0.15.0 then?

Comment (ericswanson-dfinity) [reply compatibility.json:4]: Is there an order for doing things that isn't breaking? 
If not, can we make it so by having one extension release have both filename formats?

**Review by ericswanson-dfinity — APPROVED** [review]

**Pull request merged**